decoder/src/elf2table/mod.rs

```diff
@@ -1,4 +1,4 @@
-//! Reads ELF metadata and builds a [`defmt`](https://github.com/knurling-rs/defmt) interner table.
+//! Reads ELF metadata and builds a [`defmt`](https://github.com/knurling-rs/defmt) internal table.
```

Comment:
"interner" was intentional here, but maybe we can clarify this:

```suggestion
//! Reads ELF metadata and builds a table containing [`defmt`](https://github.com/knurling-rs/defmt) format strings.
```

Reply:
Ups. Though this was a typo 🙈